### PR TITLE
Ccoss2020 fixes1803

### LINKED ISSUE #1803: [Docs] Translate Architecture page to Spanish
**Current State:**
Actual Architecture page in available in English only

**Desired State:**
Architecture page translated to Spanish.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Drake8710**
Signed-off-by: Vladimir GS <vladimir.garcia@ibm.com>

**Description**

This PR fixes #1803 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Drake8710** [comment]
Hey @jesuslerma 
Please review this. Thanks

File: docs/pages/es/architecture.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: Arquitectura de Meshery 
+permalink: es/architecture
+---
+
+## Arquitectura
+
+#### La arquitectura de Meshery puede ser vista desde dos perspectivas:
```
Comment (leecalcote) [inline docs/pages/es/architecture.md:9]: Thank you for this, @Drake8710. 

Let's avoid constraining the architectural perspective of Meshery to only two perspectives, understanding that there are a number of diagrams that we will publish over time.

File: docs/pages/es/architecture.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: Arquitectura de Meshery 
+permalink: es/architecture
+---
+
+## Arquitectura
+
```
Comment (leecalcote) [inline docs/pages/es/architecture.md:8]: Will you incorporate this new [Languages section](https://meshery.layer5.io/docs/architecture#languages), too?

File: docs/pages/es/architecture.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+#### La arquitectura de Meshery puede ser vista desde dos perspectivas:
+
```
Comment (leecalcote) [inline docs/pages/es/architecture.md:10]: Will you incorporate the [Deployments section](https://meshery.layer5.io/docs/architecture#deployments), too? 

Meshery se implementa como un conjunto de contenedores. Los contenedores de Meshery se pueden implementar en Docker o Kubernetes.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Thank you, @Drake8710 👍

**Pull request merged**